src/plugins/saved_objects_management/public/management_section/saved_objects_edition_page.tsx

```diff
@@ -42,27 +39,28 @@ const SavedObjectsEditionPage = ({
         href: '/',
       },
       {
-        text: i18n.translate('savedObjectsManagement.breadcrumb.edit', {
-          defaultMessage: 'Edit {savedObjectType}',
-          values: { savedObjectType: service?.service.type ?? 'object' },
+        text: i18n.translate('savedObjectsManagement.breadcrumb.inspect', {
+          defaultMessage: 'Inspect {savedObjectType}',
+          values: { savedObjectType: type },
         }),
       },
     ]);
-  }, [setBreadcrumbs, service]);
+  }, [setBreadcrumbs, type]);
 
   return (
-    <RedirectAppLinks application={coreStart.application}>
+    <RedirectAppLinks application={coreStart.application} style={{ height: '100%' }}>
```

Comment:
Why is specifying `height: 100%` required exactly? The underlying component seems to be using a vanilla `div`.

Reply:
Without setting a `height`, the underlying `codeEditor` isn't visible on-screen. I see other components either wrap `redirectAppLinks` in a `FlexItem` with additional styles or wrap it in a div, like we do in the [`saved_objects_table`](https://github.com/elastic/kibana/blob/master/src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx#L662).